sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/client.py

```diff
@@ -0,0 +1,1140 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+# pylint:disable=too-many-lines,too-many-public-methods
+import base64
+import uuid
+from functools import partial
+
+from azure.core.polling import LROPoller
+from azure.core.tracing.decorator import distributed_trace
+
+from ._shared import KeyVaultClientBase
+from .models import (
+    Certificate,
+    CertificateBase,
+    CertificatePolicy,
+    DeletedCertificate,
+    Issuer,
+    IssuerBase,
+    Contact,
+    CertificateOperation,
+    LifetimeAction,
+    KeyProperties,
+    KeyUsageType,
+    SecretContentType
+)
+from ._polling import CreateCertificatePoller
+
+try:
+    from typing import TYPE_CHECKING
+except ImportError:
+    TYPE_CHECKING = False
+
+if TYPE_CHECKING:
+    # pylint:disable=unused-import
+    from typing import Any, Dict, List, Optional, Iterable
+
+
+class CertificateClient(KeyVaultClientBase):
+    """CertificateClient defines a high level interface for
+    managing certificates in the specified vault.
+    Example:
+        .. literalinclude:: ../tests/test_examples_certificates.py
+            :start-after: [START create_certificate_client]
+            :end-before: [END create_certificate_client]
+            :language: python
+            :caption: Create a new ``CertificateClient``
+            :dedent: 4
+    """
+
+    # pylint:disable=protected-access
+
+    @distributed_trace
+    def create_certificate(
+            self,
+            name,  # type: str
+            policy=None,  # type: Optional[CertificatePolicy]
+            enabled=None,  # type: Optional[bool]
+            tags=None,  # type: Optional[Dict[str, str]]
+            **kwargs  # type: **Any
+    ):
+        # type: (...) -> CertificateOperation
+        """Creates a new certificate.
+
+        If this is the first version, the certificate resource is created. This
+        operation requires the certificates/create permission.
+
+        :param name: The name of the certificate.
+        :type name: str
+        :param policy: The management policy for the certificate.
+        :type policy:
+         ~azure.security.keyvault.certificates._models.CertificatePolicy
+        :param bool enabled: Determines whether the object is enabled.
+        :param tags: Application specific metadata in the form of key-value pairs.
+        :type tags: dict(str, str)
+        :returns: The created CertificateOperation
+        :rtype: ~azure.keyvault.certificates.models.CertificateOperation
```

Comment:
This returns a poller.

Reply:
Thanks for catching, switched to an LROPoller, thanks again!